CHANGELOG/CHANGELOG-3.7.md

```diff
@@ -0,0 +1,289 @@
+# Release notes for v3.7.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v3.6.0
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Enable prevent-volume-mode-conversion feature flag by default.
+  
+  Volume mode change will be rejected when creating a PVC from a VolumeSnapshot unless the AllowVolumeModeChange annotation has been set to true. Applications relying on volume mode change when creating a PVC from VolumeSnapshot need to be updated accordingly. ([#1126](https://github.com/kubernetes-csi/external-provisioner/pull/1126), [@akalenyu](https://github.com/akalenyu))
```

Comment:
This may require a major version bump:
https://kubernetes-csi.github.io/docs/project-policies.html#versioning

cc @jsafrane @gnufied @msau42

Reply:
I updated the PR to 4.0.0 first.